third_party/terraform/utils/iam_spanner_database.go

```diff
@@ -51,8 +51,7 @@ func NewSpannerDatabaseIamUpdater(d *schema.ResourceData, config *Config) (Resou
 }
 
 func SpannerDatabaseIdParseFunc(d *schema.ResourceData, config *Config) error {
-	_, err := resourceSpannerDatabaseImport("database")(d, config)
-	return err
+	return parseImportId([]string{"(?P<project>[^/]+)/(?P<instance>[^/]+)/(?P<database>[^/]+)"}, d, config)
```

Comment:
Will we need to note this as a breaking change?

Reply:
This should maintain existing behavior as far as I know, but if it doesn't, I still don't think import format changes are breaking.